Sources/SVGBezierPath.mm

```diff
@@ -68,6 +103,20 @@ + (void)resetCache
     return paths;
 }
 
++ (NSArray<SVGBezierPath*> *)pathsFromSVGString:(NSString * const)svgString viewBox:(CGRect *)viewBox
```

Comment:
why a _pointer_ to a CGRect? This creates the need to pass in an `UnsafeMutablePointer<CGRect>` in Swift.

Reply:
I used this pointer to get the viewBox info and put this value in the _svg_viewBoxCache variable. I don't know if this part can be write in a better way :)